### PR TITLE
Clean up + Python3 stuff

### PR DESCRIPTION
**Pull request by e-wipond**

### CODE REVIEW

File: 2019-todo.md [hunk 1]
```diff
@@ -1,8 +0,0 @@
-# 2019 To do
```
Comment (e-wipond) [inline 2019-todo.md:1]: Moved this into the readme.

File: STRATEGY.md [hunk 2]
```diff
@@ -1,16 +0,0 @@
-# Strategy
```
Comment (e-wipond) [inline STRATEGY.md:1]: I think this was outdated.

File: app/main.py [hunk 3]
```diff
@@ -2,12 +2,6 @@
 import os
 import random
 
-'''
```
Comment (e-wipond) [inline app/main.py:5]: Moved this into the readme.

File: requirements.txt [hunk 4]
```diff
@@ -1,4 +1,4 @@
-bottle==0.12.9
-gevent==1.0.2
-greenlet==0.4.9
-gunicorn==19.4.5
+bottle==0.12.16
```
Comment (e-wipond) [inline requirements.txt]: updated because why not

**Comment by e-wipond** [comment]
Oh yeah probably need to change the profile or runtime.txt as well.

**Comment by e-wipond** [comment]
Running here right now if you want to check it out: https://era-snake-2019-test.herokuapp.com/, I can't remember how we had it deploy setup last year, auto off of the master branch?

**Comment by Wyllan** [comment]
> Running here right now if you want to check it out: https://era-snake-2019-test.herokuapp.com/, I can't remember how we had it deploy setup last year, auto off of the master branch?

Yea, this is right. It auto deploys off master

**Review by Wyllan — APPROVED** [review]
I have python 3.7, that's the only real thing I notice. Looks good otherwise. Could maybe update the requirements to match the battlesnake repo as it is now though. Interested in the virtual environment though.

**Pull request merged**